### PR TITLE
Fix bump script for OSX

### PR DESCRIPTION
**Pull request by taminomara**
OSX `sed` is not compatible with GNU `sed`. It has different syntax for `-i` option. See this [stackoverflow] question.

[stackoverflow]: https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux

### CODE REVIEW

File: ci/bump.sh [hunk 1]
```diff
@@ -65,5 +65,9 @@ function msg {
 msg "Updating Cargo manifests with new version '$version':"
 for manifest in ethcontract*/Cargo.toml; do
 	msg "  - $manifest"
-	sed -i -E -e 's/^((ethcontract-[a-z]+ = \{ )?version) = "[0-9\.]+"/\1 = "'"$version"'"/g' "$manifest"
+	if [[ $(uname) == Darwin ]]; then
+	  sed -i '' -E -e 's/^((ethcontract(-[a-z]+)? = \{ )?version) = "[0-9\.]+"/\1 = "'"$version"'"/g' "$manifest"
```
Comment (nlordell) [inline ci/bump.sh]: ubernit: Something seems wrong with the indentation here.

**Review by nlordell — APPROVED** [review]
Minor indentation issue I think. Maybe related to tab/spaces?

File: ci/bump.sh (hunk 1 above)
Comment (taminomara) [reply ci/bump.sh]: Fixed 👍

File: ci/bump.sh [hunk 2]
```diff
@@ -65,5 +65,9 @@ function msg {
 msg "Updating Cargo manifests with new version '$version':"
 for manifest in ethcontract*/Cargo.toml; do
 	msg "  - $manifest"
-	sed -i -E -e 's/^((ethcontract-[a-z]+ = \{ )?version) = "[0-9\.]+"/\1 = "'"$version"'"/g' "$manifest"
+	if [[ $(uname) == Darwin ]]; then
+		sed -i '' -E -e 's/^((ethcontract(-[a-z]+)? = \{ )?version) = "[0-9\.]+"/\1 = "'"$version"'"/g' "$manifest"
```
Comment (taminomara) [inline ci/bump.sh:69]: Also notice that this regex was changed to update `ethcontract = ...` as well (previously it was `ethcontract-... = ...`). I totally forgot about this change when I was creating this PR 😕

**Pull request merged**